src/librustc_middle/query/mod.rs

```diff
@@ -395,8 +436,10 @@ rustc_queries! {
             desc { |tcx| "processing `{}`", tcx.def_path_str(key.to_def_id()) }
```

Comment:
This makes me realize there are pre-existing instances of `processing`, maybe open an issue about getting rid of that wording entirely, if you don't plan to handle it in this PR.